husky_components/datagrid/decorators/table-view.js

```diff
@@ -742,6 +750,7 @@ define(function() {
                 });
             }
             if (!!this.icons) {
+                content = this.addBatchesToCellContent(content, column, record);
```

Comment:
@wachterjohannes i think addBatchesToCellContent shouldn't be in if (!!this.icons) but in if (!!this.batches)